dotcom-rendering/src/components/FootballMatchStat.tsx

```diff
@@ -109,12 +147,41 @@ export const FootballMatchStat = ({
 	const homePercentage = (home.value / (home.value + away.value)) * 100;
 	const awayPercentage = (away.value / (home.value + away.value)) * 100;
 
+	const minimumContrast = 3.1; // https://www.w3.org/TR/WCAG21/#contrast-minimum
+
+	const backgroundLight = sourcePalette.neutral[97];
+	const backgroundDark = sourcePalette.neutral[10];
```

Comment:
Ideally we would get these from the palette declarations, but they're only available as custom properties so we would have to fetch the values on the client, preventing us from doing the contrast calculations during the server side render.